src/Menu.cpp

```diff
@@ -966,8 +966,8 @@ void Menu::ProcessInputEvents(RE::InputEvent* const* a_events)
 		if (it->GetEventType() != RE::INPUT_EVENT_TYPE::kButton && it->GetEventType() != RE::INPUT_EVENT_TYPE::kChar)  // we do not care about non button or char events
 			continue;
 
-		auto event = it->GetEventType() == RE::INPUT_EVENT_TYPE::kButton ? KeyEvent(static_cast<RE::ButtonEvent*>(it)) : it->GetEventType() == RE::INPUT_EVENT_TYPE::kChar ? KeyEvent(static_cast<CharEvent*>(it)) :
-		                                                                                                                                                                     KeyEvent(nullptr);  // last ternary operation should never be taken
+		auto event = it->GetEventType() == RE::INPUT_EVENT_TYPE::kButton ? KeyEvent(static_cast<RE::ButtonEvent*>(it)) : KeyEvent(static_cast<CharEvent*>(it));
+
```

Comment:
This row seems a bit weird? A lot of extra spaces?

Reply:
// last ternary operation should never be taken
and it would never be taken because there is a condition return several lines above, so this is completely unnecessary

Reply:
plus leaving a `nullptr` which will be read in the constructor without any checks is very dangerous

Reply:
I meant row 970
![image](https://github.com/user-attachments/assets/af299439-fbfb-4760-b06c-38ba51a1d22e)

Reply:
I don't see any spaces neither locally or on github

Reply:
Ok, you can resolve this then, maybe just a weird diff for me.